CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Change Log
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.4.0] - Supported Release
+### Summary
+Small release for a new feature and added support.
+
+### Features
+- Now clone your git repository as a mirror or bare repo!
+- STDERR has been added to Puppet's output
+- Added Debian 8 support
```

Comment:
Update "support" to "compatibility".

Needs to add the "Added Ubuntu 16.04 compatibility" as well.